### ISSUE TITLE
Calling the command line script failed

### ISSUE THREAD

**Issue by shadowsdb** [forlilab/Meeko#250]
我以及在conda的虚拟环境中成功安装了meeko，但是无法使用
ValueError: names passed to 'load_atom_params' need to suffixed with .json
or be the unsuffixed basename of a JSON file in /home/lishidaobiao/anaconda3/envs/env_meeko/lib/python3.9/site-packages/meeko/data/params.
name was ad4_types

**Comment by rwxayheee** [comment]
Hi @shadowsdb 
It seems like it’s because of the issue in #241 We have merged the fix but it isn’t released yet. For the time being, you can install from a download of this repository: 

```
git clone https://github.com/forlilab/Meeko.git
cd Meeko; pip install .; cd ..
```

**Comment by diogomart** [comment]
Fixed in v0.6.1, now available on PyPI and conda-forge. Thanks for reporting.

**Comment by shadowsdb** [comment]
> Hi @shadowsdb It seems like it’s because of the issue in #241 We have merged the fix but it isn’t released yet. For the time being, you can install from a download of this repository:
> 
> ```
> git clone https://github.com/forlilab/Meeko.git
> cd Meeko; pip install .; cd ..
> ```

Thank you,I have solved this problem

**Comment by shadowsdb** [comment]
> Fixed in v0.6.1, now available on PyPI and conda-forge. Thanks for reporting.

Thank you,I have solved this problem
